### ISSUE TITLE
Cyclic learning rate for transformer models

### ISSUE THREAD

**Issue by braunefe** [flairNLP/flair#2100]
Is the cyclic learning for training transformers for classification implemented in the framework?
If not, is it worth to do it and create a pull request?

**Comment by alanakbik** [comment]
Yes, you can include the cyclic learning scheduler. To train with the parameters used to train XLM, do it like this: 

```python
        # train with XLM parameters (AdamW, 20 epochs, small LR)
        trainer = ModelTrainer(tagger, corpus, optimizer=torch.optim.AdamW)
        from torch.optim.lr_scheduler import OneCycleLR

        trainer.train(
            f"path/to/your/model/folder",
            learning_rate=5.0e-6,
            mini_batch_size=4,
            mini_batch_chunk_size=1,
            max_epochs=20,
            scheduler=OneCycleLR,
            embeddings_storage_mode='none',
            weight_decay=0.,
            )
```
